recipes/onnx/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "onnx" %}
+{% set version = "0.2" %}
+{% set sha256 = "63dde5e5f3aa88a8993c65ca35960bac06ed2233001924db28a611b7c40c6254" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - protobuf
```

Comment:
This links against `libprotobuf`, right? If so, you'll need to pin the version both here and in `run` to `protobuf 3.4.*`, since protobuf [isn't backwards-compatible](https://abi-laboratory.pro/tracker/timeline/protobuf/) (see also https://github.com/conda-forge/conda-forge.github.io/pull/411).